public/static/docs/command-reference/update.md

```diff
@@ -26,15 +26,11 @@ Note that import stages are considered always locked, meaning that if you run
 `dvc repro`, they won't be updated. `dvc update` is the only command that can
 update them.
 
-Another detail to note is that when the `--rev` (revision) option of
-`dvc import` has been used to create an import stage, DVC is not aware of what
-kind of
-[Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References) this
-is, for example a branch or a tag. For typically static references (e.g. tags),
-or for SHA commits, `dvc update` will not have any effect on the import. Refer
-to the
-[re-importing example](/doc/command-reference/import#example-fixed-revisions-re-importing)
-to learn how to "update" fixed-revision imports.
+`dvc update` will not have an effect on import stages created with `dvc import`
+if option `--rev` is used with a specific commit (`rev` does not change). (In
```

Comment:
commit hash in this case better?

`(`rev` does not change)` ->  `(vs using branch or tag name)`

I like it better, but still this part "`dvc import`
if option `--rev` is used with a specific commit (`rev` does not change)"  sounds complicated.

Reply:
Hmmm yes it's a complex paragraph. I changed it to

```markdown
`dvc update` will not have an effect on import stages that are fixed to a commit
SHA hash (revision `rev` field). Please refer to
[Fixed revisions & re-importing](/doc/command-reference/import#example-fixed-revisions-re-importing)
for more details.
```
now.

Reply:
👍

Reply:
Please note that I thought we had agreed to "commit SHA hash" so I'm using that phrase here but we can review the decision for PR #552 which I'll jump right back to after this one. (It's already noted in there for when its reviewable so we won't forget.)

Reply:
I though that we agreed to use just `commit hash` after that? :) only when it's absolutely need to use the full phrase.

Reply:
I meant PR #962. The mention is in https://github.com/iterative/dvc.org/pull/962#discussion_r378059334

Reply:
Yes! But I'm leaving that change for https://github.com/iterative/dvc.org/pull/962#discussion_r378059334